UnityPy/helpers/Tpk.py

```diff
@@ -185,7 +185,7 @@ def GetDataBlob(self) -> TpkDataBlob:
         elif self.CompressionType == TpkCompressionType.Brotli:
             import brotli
 
-            decompressed: bytes = brotli.decompress(self.CompressedBytes)
+            decompressed = brotli.decompress(self.CompressedBytes)
```

Comment:
Google still hasn't merged my PR that would add type hints to brotli,
so typehinters should expect an "any" or "unknown" value for decompressed.
Either ``:bytes`` or ``= cast(bytes, ...)`` has to be used.